package_azureboard_index.json

```diff
@@ -1,475 +1,475 @@
-{

```

Comment:
did you change the file format, like line ending?

Reply:
We will close this PR, and re-update this json file